### ISSUE TITLE
The loss keeps increasing even when learning_rate is set to 0

### ISSUE THREAD

**Issue by kilickaya** [yunjey/show-attend-and-tell-tensorflow#3]
I am using the code to re-generate results, however loss increases. I wanted to debug what is going on, and set learning rate to 0, but the loss keeps increasing and exploding after even so many epochs. What could be a possible reason for this?

**Comment by yunjey** [comment]
Recently, i updated most of the code in this repo. I recommend you to reclone the repo and execute the code again then this problem will be solved.
